aptos-move/aptos-gas-calibration/README.md

```diff
@@ -56,23 +56,73 @@ public entry fun calibrate_addition(_x: u64, _y: u64) {}
 
 If the Calibration Function is expected to error, please denote it with the postfix `_should_error`.
 
+```
+//// VALID
+public entry fun calibrate_my_test_should_error() {}
+
+public entry fun calibrate_should_error() {}
+
+//// INVALID
+public entry fun should_error_calibrate() {}
+
+public entry fun calibrate_test_error() {}
+
+public entry fun calibrate_test_should_error_() {}
```

Comment:
Might be clearer if you explicitly state that these are valid calibration functions but they are not considered ones that should error?